contribs/dvrp/src/main/java/org/matsim/contrib/dvrp/path/VrpPaths.java

```diff
@@ -98,18 +123,25 @@ public static VrpPathWithTravelData createPath(Link fromLink, Link toLink, doubl
 
 		// there is no extra time spent on queuing at the end of the last link
 		links[count + 1] = toLink;
-		linkTT = getLastLinkTT(toLink, currentTime);// as long as we cannot divert from the last link this is okay
+		linkTT = travelTime.getLinkTravelTime(toLink, currentTime, null, null);
 		linkTTs[count + 1] = linkTT;
-		double totalTT = FIRST_LINK_TT + path.travelTime + linkTT;
 
+		double totalTT = (!calculateForDiversion ? FIRST_LINK_TT : 0.0) + path.travelTime + linkTT;
+		totalTT -= NODE_TRANSITION_TIME;
+		
 		return new VrpPathWithTravelDataImpl(departureTime, totalTT, links, linkTTs);
 	}
 
-	public static final double FIRST_LINK_TT = 1;
+	public static final double FIRST_LINK_TT = 2;
+	public static final double NODE_TRANSITION_TIME = 1;
 
+	/**
+	 * It would make sense to replace these calls with TravelTime.getTravelTime(...) whereever it is used /sh nov 21
+	 */
+	@Deprecated
```

Comment:
Maybe adapt this method to use `TravelTime` for the last link?

BTW. Using the travel time will lead to some overestimated travel times, but perhaps it is more accurate than assuming the free-speed times (underestimated travel times).